Automattic/jetpack
# Backup: Update review request Tracks events

#### Changes proposed in this Pull Request:

- [x] Add a new `jetpack_backup_dismiss_review_click` event when a customer dismisses a review request.

#### Jetpack product discussion


p1HpG7-i0R-p2

#### Does this pull request change what data or activity we track or use?


Yes. It adds a Tracks event to an existing link.

#### Testing instructions:





* Spin up a Jurassic Ninja site with Jetpack Beta, using Backup branch `update/backup-review-request-tracks-events`
* Purchase Backup plan with credits (or add a Jetpack Backup plan using Store Admin)
* Run a restore for your site
* Navigate to Jetpack > Backup page
* You should see the review request message
* Open developer console and execute: `localStorage.setItem('debug', 'dops:analytics');`.
* Reload/refresh the Backup page.
* Click on `Maybe later` link at the bottom of the review request component. The component should disappear and a Tracks event is fire. 
* Verify on developer console the event fires: `dops:analytics Record event "jetpack_backup_dismiss_review_click" called with props {}`
